Add Schul-Cloud

niccokunzmann opened the pull request:
Schul-Cloud is an initiative to create an open-source software for Schools around Germany.

niccokunzmann committed:
Add Schul-Cloud

Schul-Cloud is an initiative to create an open-source software for Schools around Germany.

shiftkey commented on _data/projects/schul-cloud.yml:
```diff
@@ -0,0 +1,15 @@
+name: Schul-Cloud
+desc: open-source software for schools in Germany
+site: https://github.com/schul-cloud
```
Would this link be more meaningful? https://schul-cloud.github.io/

niccokunzmann replied:
Yes, Thanks!

niccokunzmann committed:
replaced github link with github pages link

https://github.com/up-for-grabs/up-for-grabs.net/pull/596#discussion_r111904790

shiftkey commented:
@niccokunzmann thanks!
